src/packages/StorageFileApi.ts

```diff
@@ -385,9 +385,9 @@ export default class StorageFileApi {
    *
    * @param path The file path to be downloaded, including the path and file name. For example `folder/image.png`.
    */
-  getPublicUrl(path: string): string {
+  getPublicUrl(path: string): { data: { publicUrl: string }; error: null } {
```

Comment:
What do you think about not returning `error: null` at all? So just `{ data: { publicUrl: string } }`?
My thinking is it may confuse consumers if there's an error but typescript is telling them that it's always `null`

Reply:
thinking about it a bit more makes sense. Cos I assume users will do a if (error) { throw error } and typescript will complain that if (error) is false..okay changing